docs/conf.py

```diff
@@ -17,242 +17,237 @@
 import os
 from recommonmark.parser import CommonMarkParser
 
-source_parsers = {'.md': CommonMarkParser}
+source_parsers = {".md": CommonMarkParser}
```

Comment:
Sorry ran black on this file - I can restore to the one on master if you don't want those changes here!